### ISSUE TITLE
Serialize and deserialize to string helpers

### ISSUE THREAD

**Issue by petehunt** [bramstein/datrie#4]
One of the reasons this lib is helpful is for compact serializations of tries into memcached. It'd be great if I didn't have to write that serialization myself :)

**Comment by bramstein** [comment]
I agree. The current implementation does not generate compact arrays, so it isn't as useful as I hoped it would be (basically the writing and rewriting of the indexes generates a lot of empty filler space in the arrays). My plan was to rewrite it so that a object trie (say JSON, or plain old JS objects) is directly serialised into a static double array trie. I hope to have time for this soon-ish.

**Comment by monolithpl** [comment]
same problem. how about a working implementation: https://github.com/takuyaa/doublearray
